2048/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nginx:latest
+
+RUN apt-get update
```

Comment:
This and the apt-get install command should be in a single RUN instruction in order to prevent the cache from producing weird side effects.